src/components/Footer/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react'
+import cn from 'classnames'
+
+import logo from '../../assets/logo.svg'
+import sslBadge from '../../assets/ssl-badge.svg'
+
+export default class Footer extends PureComponent {
+  infoBlock = React.createRef()
+
+  render () {
+    return (
+      <div id="mc-footer">
+        <div class="mc-container mc-contain-inner">
+          <img src={logo} alt='Logo wordmark' id="logo-footer" />
+
+          <div class="nav-footer">
+            <ul class="nav-footer-items">
+              <li class="nav-footer-item">
```

Comment:
It would be great to move this to another component called `NavFooterItem`

Something like:

```javascript
const NavFooterItem = ({ label, url, external = false }) => (
  <li class="nav-footer-item">
    <a
       href={url}
       target={external ? '_blank' : '_self'}
     >
       {label}
    </a>
  </li>
)

export default NavFooterItem
```

Reply:
I'm open to pairing on this to implement, I wasn't sure how much modularity one should use in a react component - i'm still new to this!  If you're available to pair, please let me know, but for now is this ok to implement as-is?